### DISCUSSION TITLE
Cinnamon calendar app

### DISCUSSION THREAD

**Discussion by zvrabec** [linuxmint/discussions#687] [Ideas]
Hi guys!

Is there any plan to make some kind of default calendar app for cinnamon (like gnome calendar) which could connect to calendar services (like google, caldav or similar)?

Best regards!

**Comment by anaximeno** [comment]
You can already get calendar appointments from your Google account in the default calendar app by connecting your Google account in the Online Accounts settings module.

**Reply by zvrabec** [reply]
Yes, but there is no default calendar app you can manage your appointments with.

**Reply by anaximeno** [reply]
Are you on Linux Mint or another distro? Mint already comes with gnome-calendar out of the box afaik.

**Reply by zvrabec** [reply]
I'm on arch actually.
